### DISCUSSION TITLE
Isuues in stting-up a "FastLikelihood" with C9 and C10 using RK(*) measurements

### DISCUSSION THREAD

**Discussion by rajeevneutrino** [flav-io/flavio#224] [Q&A]
Dear authors,

I am interested to setup a "FastLikelihood" for my analysis with C9 and C10 from b2sll observables particularly RK, RK* and P'5.

I just followed the procedure given in https://github.com/peterstangl/flavio-lecture/blob/master/4%20Likelihoods.ipynb .
I succeeded in reproducing the same. However, if I re-define my FastLikelihood by including RK(*) observables as

FL1 = FastLikelihood(name='B2K likelihood',
                    observables=[' \<Rmue\>(B0->Kll) ', '\<Rmue\>(B+->Kll) ', '\<Rmue\>(B+->K*ll) ', ' \<Rmue\>(B0->K*ll) '],
                   )

I ended up getting this:
AssertionError: No measurement found for the observables: {' \<Rmue\>(B0->Kll) ', '\<Rmue\>(B+->Kll) ', '\<Rmue\>(B+->K*ll) ', ' \<Rmue\>(B0->K*ll) '}

But the measurements are available for all these observables. The same issue happens if I consider '\<P5p\>(B+->K*mumu)' as well.

Kindly help me if I am missing something.

Thanks & Regards
Rajeev

**Comment by DavidMStraub** [comment]
You need to provide the bins as well.

This is not a bug, converting to discussion.

**Comment by peterstangl** [comment]
@rajeevneutrino, the question of how to specify observables that require additional arguments (e.g. the bin) is answered in #225.
